website/pages/docs/plugins/simple-objects.mdx

```diff
@@ -63,23 +63,26 @@ const Node = builder.simpleInterface('Node', {
   }),
 });
 
-const UserType = builder.simpleObject('User', {
-  interfaces: [Node],
-  fields: (t) => ({
-    firstName: t.string(),
-    lastName: t.string(),
-    contactInfo: t.field({
-      type: ContactInfo,
-      nullable: false,
+const UserType = builder.simpleObject(
+    'User',
+    {
+        interfaces: [Node],
+        fields: (t) => ({
+            firstName: t.string(),
+            lastName: t.string(),
+            contactInfo: t.field({
+                type: ContactInfo,
+                nullable: false,
+            }),
+        }),
+    },
+    // You can add additional fields with resolvers with a third fields argument
+    (t) => ({
+        fullName: t.string({
+            resolve: (user) => `${user.firstName} ${user.lastName}`,
+        }),
     }),
-  }),
-  // You can add aditional fields with resolvers with a third fields argument
-  (t) => ({
```

Comment:
this was placed inside the object used as the second parameter, rather than as the third function parameter